pkg/occlient/occlient.go

```diff
@@ -876,6 +906,94 @@ func (c *Client) UpdateBuildConfig(buildConfigName string, projectName string, g
 	return nil
 }
 
+// UpdateDeploymentConfig updates the DeploymentConfig file
+// dcName is the name of the DeploymentConfig file to be updated
+// annotations contains the annotations for the DeploymentConfig file
+func (c *Client) UpdateDeploymentConfig(dcName string, annotations map[string]string) error {
```

Comment:
This is a bit confusing name. This function replaces annotations in an existing DeploymentConfig.
`UpdateDeploymentConfig` sounds like you could update anything in the DC. Let's be more descriptive in function names - `UpdateDeploymentConfigAnnotations` or just `UpdateDCAnnotations`

Reply:
ok